src/Parser/Priest/Shadow/Modules/Spells/Voidform.js

```diff
@@ -40,136 +36,158 @@ class Voidform extends Analyzer {
     return Object.keys(this._voidforms).map(key => this._voidforms[key]);
   }
 
+  get nonExcludedVoidforms(){
+    return this.voidforms.filter(voidform => !voidform.excluded);
+  }
+
+  get averageVoidformStacks() {
```

Comment:
Since we're already in the "Voidform" class, I recommend omitting the "VoidForm" from the property name; `averageStacks` would suffice. Users would likely do `this.voidform.averageStacks` anyway.